src/appMain/main.cc

```diff
@@ -55,6 +56,10 @@ int main(int argc, char* argv[]) {
 
   INIT_LOGGER(profile_instance.logs_enabled());
   CREATE_LOGGERPTR_LOCAL(logger_, "SDLMain")
+  if (!utils::appenders_loader.Loaded()) {
```

Comment:
Maybe better to implement it inside if `INIT_LOGGER` macro?
It's useful functionality for all other vendors, who can implement own `main.cc` or load SDL as dynamic library at all.